activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -136,15 +136,39 @@ def encode_with(coder)
 
     UNSAFE_STRING_METHODS.each do |unsafe_method|
```

Comment:
It seem like we should do something similar for `UNSAFE_STRING_METHODS_WITH_BACKREF`, but `gsub` and `sub` are also vararg:

```ruby
ActiveSupport::SafeBuffer::UNSAFE_STRING_METHODS_WITH_BACKREF.partition { "".method(_1).arity < 0 }
# => [["gsub", "sub"], []]
```

So it would not have an effect currently...

Reply:
Yep. That's why I left this part untouched.